test/functional/testnvim.lua

```diff
@@ -668,10 +668,11 @@ function M.set_shell_powershell(fake)
     set shellquote= shellxquote=
     let &shellcmdflag = '-NoLogo -NoProfile -ExecutionPolicy RemoteSigned -Command '
     let &shellcmdflag .= '[Console]::InputEncoding=[Console]::OutputEncoding=[System.Text.UTF8Encoding]::new();'
-    let &shellcmdflag .= '$PSDefaultParameterValues[''Out-File:Encoding'']=''utf8'';'
+    let &shellcmdflag .= '$PSDefaultParameterValues[''*:Encoding'']=''utf8'';'
+    let &shellcmdflag .= '$ErrorActionPreference=''SilentlyContinue'';'
     let &shellcmdflag .= ']] .. cmd .. [['
     let &shellredir = '2>&1 | %%{ "$_" } | Out-File %s; exit $LastExitCode'
-    let &shellpipe  = '2>&1 | %%{ "$_" } | tee %s; exit $LastExitCode'
```

Comment:
maybe someone with an windows machine can answer here.

I could not find any native tee.exe command available for windows, only an powershell alias but this is disabled some lines above, so I am wondering how this worked in tests on windows machines.

Reply:
I could not find any additional tools that are part of the agent, which would contain it. So I am currently clueless.